src/components/PopupCardDeleteVerify.js

```diff
@@ -0,0 +1,39 @@
+import Popup from "./Popup.js";
+
+export default class PopupCardDeleteVerify extends Popup {
+  constructor(popupSelector, handleFormSubmit) {
+    super(popupSelector);
+
+    this._verifyDeleteCardButton = this._popupElement.querySelector(
+      "#modal-verify-delete-card-button"
+    );
+    this._handleFormSubmit = handleFormSubmit;
+    this._handleSubmit = this._handleSubmit.bind(this);
+
+    this._popupForm = this._popupElement.querySelector(".modal__form");
+    this._submitButton = this._popupForm.querySelector(".form__button");
+    this._submitButtonText = this._submitButton.textContent;
+  }
+
+  renderLoading(isLoading, loadingText = "Saving...") {
```

Comment:
You've implemented the logic for rendering loading via the class methods. Great!